Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/EventRegistrationUIModule.kt

```diff
@@ -22,6 +24,9 @@ internal abstract class EventRegistrationUIModule {
     @ContributesAndroidInjector(modules = [ConfirmCheckInModule::class])
     abstract fun confirmCheckInFragment(): ConfirmCheckInFragment
 
+    @ContributesAndroidInjector(modules = [EditCheckInModule::class])
```

Comment:
EditCheckIn`Fragment`Module?

Reply:
id like to keep the pattern we used everywhere else